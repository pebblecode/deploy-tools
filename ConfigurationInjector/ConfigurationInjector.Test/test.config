<?xml version="1.0"?>
<configuration>
  <appSettings>
    <add key="app.environment" value="value1" />
    <add key="app.coreDbHost" value="value2" />
    <add key="app.coreDbUser" value="value3" />
  </appSettings>
</configuration>