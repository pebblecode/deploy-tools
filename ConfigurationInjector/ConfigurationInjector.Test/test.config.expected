<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <appSettings>
    <add key="app.environment" value="Production" />
    <add key="app.coreDbHost" value="db.bedegaming.com" />
    <add key="app.coreDbUser" value="gambler" />
  </appSettings>
</configuration>